preliz/predictive/predictive_explorer.py

```diff
@@ -20,8 +26,8 @@ def predictive_explorer(
     Parameters
     ----------
     fmodel : callable
-        A function with PreliZ distributions or PyMC distributions, depending on the selected
-        engine. The PreliZ distributions should call their rvs method.
+        A function with PreliZ distributions, PyMC distributions, or Bambi distributions.
+        The PreliZ distributions should call their rvs method.
```

Comment:
```suggestion
        A function with a PreliZ model, a PyMC model, or a Bambi model. See examples section below for details
```

Reply:
This could be done in a separated PR. We should add an example section to the docstring. The section should describe a minimal example using preliz, another using pymc and another bambi.